source/change-streams.rst

```diff
@@ -228,7 +228,7 @@ The driver API consists of one helper method located on a driver’s Collection
 
 The helper method must construct an aggregation command with a REQUIRED initial ``$changeStream`` stage.  A driver MUST NOT throw a custom exception if multiple ``$changeStream`` stages are present (e.g. if a user also passed ``$changeStream`` in the pipeline supplied to the helper), as the server will return an error.
 
-The aggregation command MUST use the inherited read concern of the collection it is being run against, in accordance with the `Read and Write Concern specification <https://github.com/mongodb/specifications/blob/master/source/read-write-concern/read-write-concern.rst#via-code>`_ .  The initial implementation of change streams on the server requires a “majority” read concern, drivers MUST document this requirement.  Drivers SHALL NOT throw an exception if any other read concern is specified, but instead should depend on the server to return an error.
+The aggregation command MUST use the inherited read concern of the collection it is being run against, in accordance with the `Read and Write Concern specification <https://github.com/mongodb/specifications/blob/master/source/read-write-concern/read-write-concern.rst#via-code>`_.  This MAY be overridden by a per-operation read concern if supported by the driver.  The initial implementation of change streams on the server requires a “majority” read concern, drivers MUST document this requirement.  Drivers SHALL NOT throw an exception if any other read concern is specified, but instead should depend on the server to return an error.
```

Comment:
Per @kevinAlbs' suggestion, I left the original text as-is and just added a sentence inbetween:

> This MAY be overridden by a per-operation read concern if supported by the driver.

Reply:
Maybe the wording should really just be "The aggregation command MUST use the rules for read concerns documented in the Read and Write concern specification..." since the language there already contains "In addition, a driver MAY allow it to be specified on a per-operation basis in accordance with the CRUD specification." ?

Reply:
Done. See updated commit.